tests/imap-detection/README.md

```diff
@@ -0,0 +1,10 @@
+# Simple test for imap protocol detection.
+
+## PCAP
+
+URL: https://redmine.openinfosecfoundation.org/attachments/1640 
```

Comment:
As this will point to a non-human readable file, I think we could use something like "Pcap `imap.cap` provided with redmine issue https://redmine.openinfosecfoundation.org/issues/2886"